### ISSUE TITLE
[turborepo] CLI not finding NPM package manager

### ISSUE THREAD

**Issue by vctormb** [vercel/turbo#4556]
### What version of Turborepo are you using?

1.9.1

### What package manager are you using / does the bug impact?

npm

### What operating system are you using?

Mac

### Describe the Bug

I only have NPM installed on my machine and when I run `npx create-turbo@latest myapp` the CLI says that NPM is not installed, but only pnpm.

<img width="489" alt="image" src="https://user-images.githubusercontent.com/13953703/231624620-ae5f5629-b7ff-4f59-97ca-88ccbb203602.png">

NPM and Node versions installed on my machine:
<img width="368" alt="image" src="https://user-images.githubusercontent.com/13953703/231624710-05f30755-899c-48b7-84f8-6c9159e639d9.png">

NPM Global:
<img width="367" alt="image" src="https://user-images.githubusercontent.com/13953703/231627744-25d06f81-b918-4073-8c73-b544bba5d5bb.png">

### Expected Behavior

It should be enabled only npm package manager.

### To Reproduce

Running `npx create-turbo@latest myapp` fails

Running `npx create-turbo@1.8.8 myapp` works perfectly
<img width="485" alt="image" src="https://user-images.githubusercontent.com/13953703/231628822-58cde146-9e58-4c8f-973f-b7914ec282b3.png">

**Comment by tknickman** [comment]
Got it, thanks for the detailed report! Will get this patched up today.

**Comment by dmythro** [comment]
Same issue here.
Apple M1, latest macOS, node@18 + npm@8 (no pnpm/yarn installed). Warp terminal app.

**Comment by takagimeow** [comment]
I had the same problem.
When using `create-turbo`, I was able to temporarily work around this problem by specifying `@1.8.8` instead of `@latest`.

```shell
% npx create-turbo@1.8.8

>>> TURBOREPO

>>> Welcome to Turborepo! Let's get you set up with a new codebase.

? Where would you like to create your turborepo? ./new-turborepo
? Which package manager do you want to use? (Use arrow keys)
❯ npm 
  - pnpm (not installed)
  yarn 
```

**Comment by tknickman** [comment]
Released in `create-turbo@1.9.2-canary.0`, you can test with `npx create-turbo@canary`!

**Comment by nyomansunima** [comment]
> Released in `create-turbo@1.9.2-canary.0`, you can test with `npx create-turbo@canary`!

Hey, i still faced the same issue, however the cli detect yarn and pnpm. Absolutely in my machine not installed one

**Comment by yujiym** [comment]
Hi, I still have issue with `canary`. I only install `npm` but choices are wrong.

```
npx create-turbo@canary
Need to install the following packages:
  create-turbo@1.9.2-canary.0
....
? Which package manager do you want to use?
  - npm (not installed)
❯ pnpm
  yarn
  ```

**Comment by tangirmiah** [comment]
> Hi, I still have issue with `canary`. I only install `npm` but choices are wrong.
> 
> ```
> npx create-turbo@canary
> Need to install the following packages:
>   create-turbo@1.9.2-canary.0
> ....
> ? Which package manager do you want to use?
>   - npm (not installed)
> ❯ pnpm
>   yarn
> ```

I have the same issue stated by @vctormb and using npx create-turbo@canary I get the same issue as @yujiym , I don't have pnpm or yarn installed in my machine but those are the choices I get instead of npm

**Comment by tknickman** [comment]
Reproduced the original issue with `create-turbo@1.9.2-canary.0` in an alpine container using `node@16.4.2` and `npm@7.19.1`, and validated fix using `create-turbo@1.9.2-canary.1` in the same container. 

Will get this out as `create-turbo@1.9.2` today, if anyone notices additional issues, please open a new issue! 

Thanks.
